flent/scripts/ss_iterate.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#Copyright (C) 2016  Matthias Tafelmeier
+
+#ss_iterate.sh is free software: you can redistribute it and/or modify
+#it under the terms of the GNU General Public License as published by
+#the Free Software Foundation, either version 3 of the License, or
+#(at your option) any later version.
+
+#lindwurm is distributed in the hope that it will be useful,
+#but WITHOUT ANY WARRANTY; without even the implied warranty of
+#MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#GNU General Public License for more details.
+
+#You should have received a copy of the GNU General Public License
+#along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+count=10
+interval=0.1
+host=localhost
+
+while getopts "c:I:H:t:" opt; do
+    case $opt in
+        c) count=$OPTARG ;;
+        I) interval=$OPTARG ;;
+        H) host=$OPTARG ;;
+        t) target=$OPTARG ;;
```

Comment:
They're not in interactive use, no, but they have defaults for all parameters. 

What I want to achieve here is that it should be obvious why the script fails if it does. And if it's just outputting grep errors, that is not obvious. I think that is less of a concern with the ss filtering, but I'd like to be fairly certain that a debug log (which contains the command output prior to parsing) gives a useful clue if something breaks...